src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -41,19 +41,9 @@
             </small>
           </div>
 
-          <div class="form-group time-and-pattern">
-            <label>
-              <input
-                data-test-subj="createIndexPatternIsTimeBasedCheckBox"
-                ng-model="controller.newIndexPattern.isTimeBased"
-                type="checkbox">
-              <span translate="KIBANA-CONTAINS_TIME_BASED_EVENTS"></span>
-            </label>
-          </div>
-
-          <div class="form-group" ng-if="controller.newIndexPattern.isTimeBased">
+          <div class="form-group">
             <label>
-              <span translate="KIBANA-TIME_FIELD_NAME"></span>
+              <span translate="KIBANA-TIME_FILTER_FIELD_NAME"></span>
```

Comment:
Hmmm... I guess "time filter" works. I like "time picker" because I know what that is, but I'm afraid that might couple these field too closely to a particular UI component.